app/workers/process_photo.rb

```diff
@@ -15,6 +15,15 @@ def perform(id)
 
       photo.processed_image.store!(unprocessed_image)
 
+      ##unless current_user.strip_exif
+		  exifdata = MiniExiftool.new unprocessed_image.path, :coord_format => '%+.18f'
```

Comment:
- Tab detected.
- Inconsistent indentation detected.
- Use the new Ruby 1.9 hash syntax.
- Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.